.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  global:
+    - LUAJIT_LIB=/usr/lib64/libluajit-5.1.so
+    - LUAJIT_INC=/usr/include/luajit-2.1
+    - LUA_INCLUDE_DIR=/usr/include/luajit-2.1
+    - LUA_CMODULE_DIR=/lib
+  matrix:
+    - NGINX_VERSION=1.9.15
+    
+before_install:
+  - sudo apt-get install -qq -y axel cpanminus > build.log 2>&1 || (cat build.log && exit 1)
```

Comment:
"apt-get install -qq" is pretty silent itself, I wouldn't add output redirection here